src/app/util/chip-response.cpp

```diff
@@ -25,13 +25,20 @@
 #include <assert.h>
 #include <inet/InetLayer.h> // PacketBuffer and the like
 #include <support/logging/CHIPLogging.h>
+#include <transport/SecureSessionMgr.h> // For SecureSessionMgrBase
 
 using namespace chip;
 
+// This is not ideal, but we're still sorting out how secure session managers
+// end up working and whether they're singletons.  In the long term, there will
+// be some sane API that lets us send a message to a given node id.
```

Comment:
Can you add the string "TODO" in here somewhere.

Reply:
Yep, done.

Reply:
And https://github.com/project-chip/connectedhomeip/issues/2566 filed.